infra/ansible/roles/django/handlers/main.yml

```diff
@@ -45,14 +45,6 @@
   become: true
   become_user: gunicorn
 
-- name: Populate Django DB from Public Dataset
```

Comment:
I got an error while trying this PR, and I realized that this task is still notified by another task in the django recipe. We can remove it from the notify list.



```yaml
# file: `infra/ansible/roles/django/tasks/main.yml`

- name: Copy Django OIDC RSA private key
  template:
    src: django_oidc_rsa_private_key.j2
    dest: /root/django_oidc_rsa_private_key
    mode: u=rw,g=,o=
  notify:
    - Populate Django DB from Public Dataset
    - Create Swagger UI OAuth application in Django database
 ```

Reply:
Fixed by https://github.com/tournesol-app/tournesol/pull/1252/commits/597cc79643acd5a3d7fd71790f5368ba8fe20fee